### PR TITLE
Grammatically updated the README file

### PR DESCRIPTION
**Pull request by avinash-ak**
##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by avinash-ak** [comment]
@anotherjesse

**Comment by avinash-ak** [comment]
@bcoca Please review the PR and merge if it is correct. Thanks :)

File: README.rst [hunk 1]
```diff
@@ -12,7 +12,7 @@ changes like zero-downtime rolling updates with load balancers easy. More inform
 Design Principles
 =================
 
-*  Have a dead simple setup process and a minimal learning curve.
+*  Have a dead-simple setup process with minimal learning curve.
```
Comment (samccann) [inline README.rst]:
```suggestion
*  Have a dead-simple setup process with a minimal learning curve.
```

**Comment by samccann** [comment]
Hi @avinash-ak and thanks for helping out with Ansible!

Looking over this change, it seems the original was correct 'with a minimal learning curve'.  I put that in as a suggested edit.  
In case GitHub is new to you, you should see something like the following:

image

You can just click 'commit suggestion' to accept this change.

**Comment by avinash-ak** [comment]
Hi @samccann i have committed the suggestion. Could you please merge the PR if it is fine now. Thanks Sandra 👍

**Review by samccann — APPROVED** [review]

**Comment by samccann** [comment]
Thanks @avinash-ak and welcome to Ansible!  If you are looking for other ways to continue in the Ansible community, check out some of the ideas here - https://docs.ansible.com/ansible/latest/community/index.html .

**Pull request merged**